src/WarcraftLegacies.Source/Quests/Legion/QuestAlteracBase.cs

```diff
@@ -16,19 +18,19 @@ public sealed class QuestAlteracBase : QuestData
     /// <summary>
     /// Initializes a new instance of the <see cref="QuestAlteracBase"/> class
     /// </summary>
-    public QuestAlteracBase(Rectangle rescueRect) : base("Ruins of Alterac",
-      "The orcs that occupied Alterac have maintained a secret demon gate, the Legion will make good use of it",
+    public QuestAlteracBase(Rectangle rescueRect, Capital caerDarrow) : base("Ruins of Alterac",
+      "The orcs that occupied Alterac have maintained a secret demon gate, the Legion will make good use of it. Thanks to their orc servants, the legion was able to propagate grim shrines around Lordaeron.",
```

Comment:
The Description field should describe the current situation, not what's happened as a result of the quest, since it hasn't been completed yet.

Reply:
The more I think about it the more I think this quest just needs a total rewrite to account for the fact that the objective is _actually_ just to complete the Plague. I'll make another ticket and we can mark this as resolved.